community_supplied/Interface/oc-interface-throughput.rule

```diff
@@ -1,11 +1,350 @@
+                }
+                /*
+                * Defaults color to green.
+                */
+                term is-input-bandwidth-normal {
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name input bandwidth utilization ($in-bw-util %) is normal. Mbps:$in-mbps Input bytes:$input-bytes";
+                        }
+                    }
+                }
+            }
+            trigger out-bw-utilization {
+                synopsis "Interface output bandwidth utilization ";
+                description "Sets health based on the change in output bandwidth utilization";
+                frequency 90s;
+                /*
+                * Sets color to red and sends out an anomaly notification when
+                * the output bandwidth utilization ($out-bw-util %) is greater than
+                * the threshold ($threshold %).
+                */
+                term is-output-bandwidth-abnormal {
+                    when {
+                        greater-than-or-equal-to "$out-bw-util" "$threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name output bandwidth utilization($out-bw-util%) crossed threshold value ($threshold %). Mbps:$out-mbps Output bytes:$output-bytes";
+                        }
+                    }
+                }
+                /*
+                * Sets color to yellow and sends out an anomaly notification when
+                * the dynamic threshold ($dt-out-bw-util) of output bandwidth
+                * utilization is equal to 1.
+                */
+                term is-output-bandwidth-above-dt {
+                    when {
+                        equal-to "$dt-out-bw-util" 1;
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "$interface-name output bandwidth utilization ($out-bw-util %) is above dynamic threshold. Mbps:$out-mbps Output Bytes:$output-bytes Predicted BW utilization:$predicted-out-bw-util";
+                        }
+                    }
+                }
+                /*
+                * Defaults color to green.
+                */
+                term is-output-bandwidth-normal {
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name output bandwidth utilization ($out-bw-util %) is normal. Mbps:$out-mbps Output bytes:$output-bytes";
+                        }
+                    }
+                }
+            }
+            /*
+            * Variables with default values. Default values can be changed
+            * while deploying playbook instance.
+            */
+            variable in-stats-name {
```

Comment:
No need of templating in, out octets as it is bw rule hence remove in-octets and out-octets variable.